pywb/apps/rewriterapp.py

```diff
@@ -854,6 +854,8 @@ def handle_query(self, environ, wb_url, kwargs, full_prefix):
             'url': wb_url.url,
         }
         res = self.do_query_timeline(wb_url, kwargs)
+        if not res.status_code == 200:
```

Comment:
Can we test this against revisits (ie. URLs that may return a status code of 301/302?)